contribution-guide.md

```diff
@@ -0,0 +1,87 @@
+
+
+# Contribution guide
+
+## What do I need to know to help?
+There can be multiple ways in which you can help us improve the project from what it is today. Before you begin, we would encourage you to start exploring the extension, to understand more about the project. Once you are done, you can begin contributing with code, with content or simply by giving us ideas.
+### Contributing code
+The chrome extension is built using with Node.js on the top of HTML , CSS and JavaScript and JQuery. Other than that, we use git for version control, so knowledge of git would come handy.
+### Contributing content / documentation
+In order to add documentation for the project, you need to have some understanding about the project, and decent technical writing skills.
+However, if you do not understand certain things about the project, the community will always be there to help you.
+### Finding Bugs / Giving feature ideas
+This is the easiest way of helping us with the project. You are encouraged to try out the latest development version, explore different features, and find out if there are any bugs. You can also brainstorm feature ideas, and share them with us.
+To share either bug or features, [create an issue here](https://github.com/internetarchive/wayback-machine-chrome/issues/new/choose), and we can continue the rest of the discussion there.
+
+## How to set up the development environment?
+Before we begin, if you don't already have, download [git](https://git-scm.com/downloads) and [node.js](https://nodejs.org/en/download/) (You may look at relevant tutorials to set them up according to your operating system)
```

Comment:
Please specify which NODE Version

Reply:
@anishsarangi which version is it exactly?

Reply:
I am not sure completely at this moment !

Reply:
@cgorringe @tikhsuP Which node version is it?

Reply:
I don't understand why a node.js version needs to be mentioned, since it will obviously change over time, unless there's a requirement that it must be >= a minimum version? Wouldn't saying to download the "latest" version suffice?

Reply:
@cgorringe I remember there was a specific node version there while creating this extension. I think we have included it in Readme.md, so no need to include here.